Automattic/wp-calypso
# Devdocs: Remove experimental WordPress components

#### Changes proposed in this Pull Request

Remove devdocs for experimental `@wordpress/components`. We should not encourage usage of experimental APIs.

See p4TIVU-9xW-p2#comment-10759

#### Testing instructions

* Devdocs should build and compile.
* Experimental components should not be available:  https://calypso.live/devdocs/wordpress-components-gallery?branch=remove/devdocs-wp-experimental-apis
* Lint should pass. 
* Full lint for this rule `npx eslint-nibble --ext .js,.jsx,.ts,.tsx --no-interactive --rule wpcalypso/no-unsafe-wp-apis .` should report no errors.
